activation/identities.go

```diff
@@ -29,5 +29,5 @@ func (s *IdentityStore) StoreNodeIdentity(id types.NodeId) error {
 func (s *IdentityStore) GetIdentity(id string) (types.NodeId, error) {
 	key := getKey(id)
 	bytes, err := s.ids.Get(key[:])
-	return types.NodeId{id, bytes}, err
+	return types.NodeId{Key: id, VRFPublicKey: bytes}, err
```

Comment:
I don't like this asymmetry between "key" and "VRFpublickey"; a better name is "VRFkey".
"EdwardsKey" may be a solution, though it is also not that great.

Reply:
I agree. Specifically, this change is just adding the field names to the assignment, since it's better practice than assuming that the order of fields will remain constant. Let's keep what you're suggesting for a future PR.